dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.socialLogin1.0.internal.ee-10.0.feature

```diff
@@ -8,5 +8,5 @@ visibility = private
   io.openliberty.security.social.internal,\
   io.openliberty.security.openidconnect.internal.clients.common,\
   io.openliberty.security.oidcclientcore.internal.jakarta
-kind=noship
-edition=full
+kind=beta
```

Comment:
Why are we moving this feature to beta as part of the Servlet 6.0 beta work?

Reply:
reverted the change

Reply:
To answer the question, the reason I told Phu to also update this one is because servlet-6.0 was the final needed feature to move to beta in order to move this feature to beta.